src/components/date-picker-day/date-picker-day.scss

```diff
@@ -221,7 +221,7 @@
 }
 
 :host([highlighted]:not([active]:focus)) .day {
-  @apply text-color-1 rounded-none;
+  @apply text-color-1 rounded-full;
```

Comment:
I don't think `rounded-full` is required here.

Reply:
This change is what fixes the bug.

Reply:
we don't need to add `rounded-full` here as it is the default style applied on `.day`